wiki/Difficulties/ja.md

```diff
@@ -0,0 +1,71 @@
+---
+tags:
+  - difficulty
+  - difficulty name
+  - SR
+  - star rating
+  - stars
```

Comment:
these should be written in japanese. they don't have to be translated from en but they should help people search for this article on https://osu.ppy.sh/home/search?mode=wiki_page

Reply:
Okay. Translate them.